src/util/RequestsUtil.ts

```diff
@@ -93,4 +95,32 @@ export class RequestsUtil {
 		}
 		return this.hashCode( resolver.tag ) & 0x00FFFFFF;
 	}
+
+	public static async checkTicketValidity( ticketKeyString: string ): Promise<boolean> {
+		try {
+			const searchResults = await MojiraBot.jira.issueSearch.searchForIssuesUsingJqlGet( {
+				jql: `(${ BotConfig.request.invalidRequestJql }) AND key in (${ ticketKeyString })`,
```

Comment:
Would be nice to have different invalid JQLs for different channels; I might come back to that later. Out of scope for this PR though.